FutureCollars/src/main/java/lesson5task1/Text.java

```diff
@@ -0,0 +1,31 @@
+package lesson5task1;
+
+import java.util.Scanner;
+
+public class Text {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Hello world");
+        String text = scanner.nextLine();
+
+        if (text.length() == 0) {
+            System.out.println("Empty");
+        } else {
+            int length = text.length();
```

Comment:
te zmienną możesz stworzyć przed ifem i wykorzystać też we wcześniejszym warunku. W ten sposób unikniesz duplikacji użycia metody text.length() :)